modules/apps/portal-security/portal-security-ldap-impl/src/main/java/com/liferay/portal/security/ldap/internal/authenticator/LDAPAuth.java

```diff
@@ -375,48 +386,42 @@ private int _authenticate(
 			LDAPAuthResult ldapAuthResult = _authenticate(
 				safeLdapContext, companyId, attributes, fullUserDN, password);
 
-			// Get user or create fromUnsafe LDAP
-
 			if (!ldapAuthResult.isAuthenticated()) {
 				password = null;
```

Comment:
This line will never have an effect as we will return `FAILURE` before reaching `_ldapUserImporter.importUser(ldapServerId, companyId, safeLdapContext, attributes, password);`